server/controller/src/main/java/ai/starwhale/mlops/domain/ft/FineTuneAppService.java

```diff
@@ -301,20 +306,45 @@ private FineTuneVo buildFineTuneVo(FineTuneEntity fineTuneEntity) {
                 .build();
     }
 
-    public int importEvalFromCommon(Long projectId, Long spaceId, List<String> uuids) {
-        return evaluationRepo.migration(
+    public MigrationResult importEvalFromCommon(Long projectId, Long spaceId, List<String> uuids) {
+        // validate the src rows
+        // 1. status should be final
+        var jobs = jobMapper.findJobByUuids(uuids);
```

Comment:
Find jobs with uuid and `projectId` will be better.

Reply:
done